dashboard/app/main.go

```diff
@@ -490,14 +492,17 @@ func textFilename(tag string) string {
 	}
 }
 
-func fetchNamespaceBugs(c context.Context, accessLevel AccessLevel, ns string) ([]*uiBugGroup, int, error) {
+func fetchNamespaceBugs(c context.Context, accessLevel AccessLevel, ns string, manager string) ([]*uiBugGroup, int, error) {
 	var bugs []*Bug
 	_, err := db.NewQuery("Bug").
 		Filter("Namespace=", ns).
 		GetAll(c, &bugs)
 	if err != nil {
 		return nil, 0, err
 	}
+	if len(manager) != 0 {
```

Comment:
`manager != ""` is a more check canonical for strings

Reply:
Fixed.